tests/utils.py

```diff
@@ -24,6 +24,30 @@ def fig2array(fig):
     return data
 
 
-def match_figure_image(fig, arr_path):
-    """Check whether the two image arrays match."""
+def match_figure_image(fig, arr_path: str):
+    """Check whether the two image arrays match.
+
+    Args:
+        fig (:class:`matplotlib.figure.Figure`): Figure to compare.
+        arr_path (str): Path to the ``numpy`` array file containing the reference image.
+
+    Returns:
+        bool: True if the images match, False otherwise.
+    """
     return np.all(fig2array(fig) == np.load(arr_path))
+
+
+def match_figure_close_image(fig, arr_path, rtol=0, atol=255):
+    """
+    Check whether the two image arrays match within a tolerance.
+
+    Args:
+        fig (matplotlib.figure.Figure): Matplotlib figure to compare.
+        arr_path (str): Path to the numpy array file containing the reference image.
+        rtol (float, optional): Relative tolerance for comparison.
+        atol (float, optional): Absolute tolerance for comparison.
+
+    Returns:
+        bool: True if the images match within the specified tolerances, False otherwise.
```

Comment:
```suggestion
def match_figure_close_image(fig, arr_path: str, rtol: float = 0, atol: float = 255):
    """Check whether the two image arrays match within a tolerance.

    Args:
        fig (:class:`matplotlib.figure.Figure`): Figure to compare.
        arr_path (str): Path to the ``numpy`` array file containing the reference image.
        rtol (float, optional): Relative tolerance for comparison.
        atol (float, optional): Absolute tolerance for comparison.

    Returns:
        bool: ``True`` if the images match within the specified tolerances, ``False`` otherwise.
```